src/data-tutorials/users.tsx

```diff
@@ -761,6 +761,13 @@ const Users: User[] = [
     website: 'blog/tutorials/como-implantar-um-contrato-inteligente-ERC721-usando-a-API-Tatum',
     tags: ['intermediate','celosage','erc721','smartcontract','video'],
   },
+  {
+    title: 'Creating Smart Contracts for liquidity farming',
+    description: ' Describing Smart Contracts For Liquidity Farming And The Technical Side Of Creating Them.',
+    preview: require('./showcase/intermediate/creating-smart-contracts-for-liquidity-farming.png'),
```

Comment:
./showcase/intermediate/creating-smart-contracts-for-liquidity-farming.png

There is no Image with This Name

Reply:
@Manuel-dre  You Can Change your Images Name like celo-smart-contracts.jpg